bench-tps/src/spl_converters.rs

```diff
@@ -0,0 +1,50 @@
+use {
```

Comment:
Looks like account-decoder crate is the best candidate, but I'm not sure if moving there instructions converters is aligned with the name "account-decoder". It looks like it would have more sense to have a more generic crate with all types of spl-converters. Might be a separate issue, what do you think?

Reply:
> 

These macroses might be a good thing to add. I don't know if they are too heavy to be part of this PR or not, feel free to proceed with this PR the way you think is optimal (including macroses in PR or move to the following PR/issue).